### ISSUE TITLE
The APK for the app? 🍩 

### ISSUE THREAD

**Issue by nimit2801** [jxareas/Xpensor#22]
- I found the app here: https://education.github.com/globalcampus/student
- Wanted to know if this was ship ready
- If yes, how can we directly use this app, without building it from scratch :)

**Comment by jxareas** [comment]
Hi.
Certainly, the app is not yet finished, as there is some core functionality still being worked on, although most of the basic features are already completed (add account, add transaction, filter by date, filter by account, show pie chart, etc).

Anyways, once it is done, I'll publish the .apk in the releases section, so that it is not necessary for you to build it via Android Studio.

**Comment by nimit2801** [comment]
> Hi. Certainly, the app is not yet finished, as there is some core functionality still being worked on, although most of the basic features are already completed (add account, add transaction, filter by date, filter by account, show pie chart, etc).
> 
> Anyways, once it is done, I'll publish the .apk in the releases section, so that it is not necessary for you to build it via Android Studio.

Yes sure that would be great. Let me know if there are some issues open to work upon :)
Would be glad to help you out ✅ 

Thank you

**Comment by IzzySoft** [comment]
And if you think it's ready for an "open audience": will you start tagging releases here and attach the APK there, so it can be found easily?

**Comment by jxareas** [comment]
I definitely will. Thanks for the feedback!

**Comment by IzzySoft** [comment]
Great! Please drop a note here in this issue when the APK is there :star_struck:

**Comment by jxareas** [comment]
I apologize for the late response but I've finally attached the debug build in the first [Alpha release](https://github.com/jxareas/Xpensor/releases/tag/v0.1.0-alpha), so that it can be downloaded easily.
Despite being a pre-release it contains most of the application logic. The remaining finishing features will be added soonly 🚀.

**Comment by IzzySoft** [comment]
Thanks a lot! How about a release build, with [properly set versioning](https://github.com/jxareas/Xpensor/blob/e0c6c4f9ac7883deb95890ad29cb4eba9ac6e97f/buildSrc/src/main/java/ProjectProperties.kt#L15)? The current one, despite being v0.1.0-alpha, identifies as

    package: name='com.jxareas.xpensor' versionCode='1' versionName='1.0'

which not only is a bit confusing, but also makes it hard for my updater to check for a newer version (it would ignore all tags pointing to a version <= 1.0, as it thinks they are older).

**Comment by jxareas** [comment]
Whoops! 😅 Yeah you are right, I was migrating the Gradle files from Groovy to KTS and actually forgot to update the version in the `ProjectProperties.kt` file. Thanks for reminding me! I appreciate it a lot and will correct it ASAP.

**Comment by IzzySoft** [comment]
Oh, already done – and there I wait for a ping :see_no_evil: Set it up then for my repo, should become [visible here](https://apt.izzysoft.de/fdroid/index/apk/com.jxareas.xpensor) with the next sync at 7 pm UTC. Be welcome to [pick a badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) if you want to link there form e.g. your readme :smiley:

For now it's clearly marked "alpha" and "debug"; as soon as you provide release builds, the warning will be removed.

Thanks again!

**Comment by jxareas** [comment]
Will do, I appreciate the info! [I've also updated the previous release](https://github.com/jxareas/Xpensor/releases/tag/v0.1.0-alpha) with the *build variant .apk*.

**Comment by IzzySoft** [comment]
I usually don't replace APKs which have already been published – but that can of course be done. Though it would require the replacement to be properly signed :wink:

```
$ apksigner verify --verbose xpensor-release.apk
DOES NOT VERIFY
ERROR: Missing META-INF/MANIFEST.MF
```

If you're planning a new release within a week or two, it would fully suffice to wait for that. If not, and there's a properly signed release build, I can replace the existing APK – which would only affect new installs of course.
